code/stats/scoring.cpp

```diff
@@ -105,12 +105,10 @@ void parse_rank_tbl()
 		required_string("#End");
 
 		// be sure that all rank points are in order
-#ifndef NDEBUG
```

Comment:
No, the content of `Warning` is entirely empty in [dialogs.cpp](https://github.com/scp-fs2open/fs2open.github.com/blob/master/code/osapi/dialogs.cpp#L448) if we are in release mode.